package.json

```diff
@@ -17,7 +17,7 @@
     "diff": "~1.2.2",
     "dropzone": "git://github.com/sloria/dropzone.git#accept-directory",
     "exports-loader": "^0.6.2",
-    "express": "~4.10.0",
+    "express": "^4.10.8",
```

Comment:
Better to pin to an exact version if possible rather than using `^`. If not possible due to a conflicting dependency, use `~`. See: http://stackoverflow.com/a/22345808